src/components/Card.js

```diff
@@ -1,11 +1,15 @@
 import { cardselectors } from "../utils/constants.js";
 
 export default class Card {
-  constructor(data, cardSelector, handleImageClick) {
+  constructor(data, cardSelector, handleImageClick, handleDeleteCard, handleLikeIcon) {
     this._name = data.name;
     this._link = data.link;
+    this._id = data._id;
+    this._isLiked = data.isLiked;
```

Comment:
`.setLike(cardData.id, cardData.isLiked)`

To make it work you need to change them in class `Card`. 

`this._id` should be `this.id` and `this._isLiked` should be `this.isLiked`